### PR TITLE
{bio}[foss/2020b] mapDamage v2.2.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.5
See https://gist.github.com/f32832f3c570d46fe0cdc7111690bc94 for a full test report.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/03ad8889ea432cab39c2f2f67e81bb0d for a full test report.

File: easybuild/easyconfigs/m/mapDamage/mapDamage-2.2.1-foss-2020b-R-4.0.4.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'mapDamage'
+version = '2.2.1'
+versionsuffix = '-R-%(rver)s'
+
+github_account = 'ginolhac'
+
+homepage = 'https://ginolhac.github.io/mapDamage/'
+description = '''mapDamage2 is a computational framework written in Python and
```
Comment (verdurin) [inline easybuild/easyconfigs/m/mapDamage/mapDamage-2.2.1-foss-2020b-R-4.0.4.eb:10]: <nitpick> - shouldn't this be `"""`?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/mapDamage/mapDamage-2.2.1-foss-2020b-R-4.0.4.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/m/mapDamage/mapDamage-2.2.1-foss-2020b-R-4.0.4.eb]: Thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.5
See https://gist.github.com/e3e56af0a9e7db63e81d929496f1f679 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @akesandgren!

**Pull request merged**